.github/ISSUE_TEMPLATE/report-a-bug.yml

```
@@ -0,0 +1,74 @@
+name: Report bug
+description: Report a bug in journal-policy-tracker-backend.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report an journal-policy-tracker-backend bug! Fill out the form below to provide us with info to help fix the bug.
+        Only use this if you're 100% sure you've found a bug and can reproduce it.
+
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: Describe roughly what the bug is here.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Provide an example of how to trigger the bug.
+      placeholder: |
+        Example:
+        1. do this 
+        2. after this do that
+        3. you get the gist...right?
+    validations:
+      required: true
+  
+  - type: textarea
+    attributes:
+      label: Expected behaviour
+      description: Explain what you should expect to happen.
+      placeholder: |
+        Example: "This is the expected behaviour from it."
```

What do you think about rewording like this?

```suggestion
        Example: "It was expected that <X> happened."
        Or "Data returned should be in format <Y>."
```